client/src/components/CardDialog/CheckList/CheckList.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import Input from '@material-ui/core/Input';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import ListItemText from '@material-ui/core/ListItemText';
+import Select from '@material-ui/core/Select';
+import Checkbox from '@material-ui/core/Checkbox';
+import checkListStyles from './checkListStyles';
+
+const ITEM_HEIGHT = 48;
+const ITEM_PADDING_TOP = 8;
+const MenuProps = {
+  PaperProps: {
+    style: {
+      maxHeight: ITEM_HEIGHT * 4.5 + ITEM_PADDING_TOP,
+      width: 250,
+    },
+  },
+};
+
+const names = ['Buy supplies', 'Contact teammates', 'Research', 'Book study hours'];
```

Comment:
`const NAMES = ...` use this for constants instead of lowercase ([best practice](https://www.javascripttutorial.net/es6/javascript-const/#:~:text=The%20const%20keyword%20creates%20a%20read%2Donly%20reference%20to%20a%20value.&text=By%20convention%2C%20the%20constant%20identifiers,keyword%20can't%20be%20reassigned.))

Reply:
Noted. I'm aware of this convention for Python, but am not as well versed on JS/TS variable naming so it's appreciated.